### DISCUSSION TITLE
Tracking when LanguageStatusItem is pinned

### DISCUSSION THREAD

**Discussion by Bernardin-MS** [microsoft/vscode-discussions#216] [Extension Development QnA] [answered]
Hi,

We're working on the C++ extension and the LanguageStatusItem feature is being added. I wanted to know if there was some signal that fired when a LanguageStatusItem has been pinned. We wanted to be able to change the text to a shorthand version when the item is pinned.

If that isn't possible could some kind of pinnedText attribute be added?

Thanks

**Comment by isidorn** [comment, marked as answer]
I believe there is no API fire event when a `LanguageStatusItem` is pinned. @jrieken can correct me if needed.
